truck_delivery_atge/client_manager.py

```diff
@@ -0,0 +1,26 @@
+
+from truck_delivery_atge.content_manager import ContentManager
+from truck_delivery_atge.db_connector import DBConnector
+from truck_delivery_atge.client import Client
+
+CLIENTE_ID = "UUI-client"
```

Comment:
you should be importing the UUI from the utils module

Reply:
added